CHANGELOG.md

```diff
@@ -30,28 +33,45 @@ be found.
 
 + Syslog logging driver now supports DGRAM sockets [#21613](https://github.com/docker/docker/pull/21613)
 + Add `--details` option to `docker logs` to also display log tags [#21889](https://github.com/docker/docker/pull/21889)
++ Enable syslog logger to have access to env and labels [#21724](https://github.com/docker/docker/pull/21724)
++ Additional syslog-format option to allow microsecond resolution in syslog timestamp [#21844](https://github.com/docker/docker/pull/21844)
```

Comment:
**An** additional syslog-format option **(`rfc5424micro`) has been added** to allow microsecond resolution in syslog timestamp ?